core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -220,8 +219,8 @@ protected void tryComplete() {
 
     synchronized (this) {
       if (state == STATE_RESPONSE && allChildrenComplete) {
-        if (LOGGER.isDebugEnabled()) {
-          LOGGER.debug(this + " completed.");
+        if (debugLogEnabled) {
+          LOGGER.debug("{} completed.", this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: No need to call "toString()" method as formatting and string conversion is done by the Formatter. ([squid:S3457](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3457))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLjRWXotIXnmsUmglB&open=AWkLjRWXotIXnmsUmglB)

[ISSUE_KEY]: AWkLjRWXotIXnmsUmglB